packages/ngx-deploy-npm/package.json

```diff
@@ -10,8 +10,10 @@
   "engines": {
     "node": ">=14.0.0"
   },
-  "dependencies": {
-    "@nrwl/devkit": "15.6.2"
+  "peerDependencies": {
+    "@nrwl/devkit": "^15.0.0",
+    "@nrwl/workspace": "^15.0.0",
```

Comment:
Added this one, I noticed in the source code that it was used. Not sure why it hadn't failed before.

Reply:
You're right!

Like many other nx plugins, I created the `init` generator to install the dependencies ngx-deploy-npm would need. Right now, it uses none. So that `init` generator is useless. That's why it wasn't failing.

Let's go ahead and leave it. It might get removed in the future.